pubspec.yaml

```diff
@@ -11,4 +11,4 @@ dev_dependencies:
   benchmark_harness: ">=1.0.5 <2.0.0"
   matcher: ">=0.12.0 <0.13.0"
   test: ">=0.12.30 <1.4.0"
-
+  pedantic: ^1.9.0
```

Comment:
Should I open a PR to add this dependency? When I first cloned this repo I was greeted with a lot of errors due to this package not being present which was confusing.